backend/src/main/java/app/linguistai/bmvp/controller/AccountController.java

```diff
@@ -71,7 +78,7 @@ public ResponseEntity<Object> changePassword(@Valid @RequestBody QChangePassword
     public ResponseEntity<Object> refreshToken(@RequestHeader(HttpHeaders.AUTHORIZATION) String auth) {
         try {
             RRefreshToken newToken = accountService.refreshToken(auth);
-            return Response.create("new accsess token is created", HttpStatus.OK, newToken);
+            return Response.create("new access token is created", HttpStatus.OK, newToken);
         } catch (Exception e) {
             return Response.create(ExceptionLogger.log(e), HttpStatus.BAD_REQUEST);
```

Comment:
Keske exceptionlarimizdan gusel gusel error mesajlari donse :(

Reply:
Complete exception overhaul in another PR